packages/webapp/src/store/api/apiSlice.ts

```diff
@@ -192,6 +192,14 @@ export const api = createApi({
       }),
       invalidatesTags: ['AnimalBatches', 'CustomAnimalTypes', 'CustomAnimalBreeds'],
     }),
+    updateAnimals: build.mutation<void, Partial<Animal>[]>({
+      query: (body) => ({ url: `${animalsUrl}`, method: 'PATCH', body }),
+      invalidatesTags: ['Animals', 'CustomAnimalTypes', 'CustomAnimalBreeds'],
+    }),
+    updateAnimalBatches: build.mutation<void, Partial<AnimalBatch>[]>({
+      query: (body) => ({ url: `${animalBatchesUrl}`, method: 'PATCH', body }),
+      invalidatesTags: ['AnimalBatches', 'CustomAnimalTypes', 'CustomAnimalBreeds'],
+    }),
```

Comment:
Yes! The numbers should be updated, great catch

Reply:
We had a ticket with a bug report for this, and I forgot there was already a comment for this here when I grabbed it 😅 but I did find I think a better way to get the KPIs updated, which invalidates only the request for the count instead of re-fetching all of the default types themselves. That should be a little bit more performant, since one of the pros of separating default and custom types was that we wouldn't need to re-fetch default types constantly since they're somewhat static. No need to change anything here though, I already have a branch for this so I'll be opening a PR soon

Reply:
My solution ended up not making sense, so I'm adding the Jira link for the bug here so that once this one's merged it moves the ticket to Validation!